src/data/polynomial/eval.lean

```diff
@@ -779,6 +784,18 @@ begin
   exact ring_hom.map_zero f,
 end
 
+lemma is_root.map {f : R →+* S} {x : R} {p : polynomial R} (h : is_root p x) :
+  is_root (p.map f) (f x) :=
+by rw [is_root, eval_map, eval₂_hom, is_root.def.1 h, f.map_zero]
```

Comment:
It might be nice to add `is_root.eq_zero (h : is_root p x) : eval x p = 0 := h`, which you can then use to very slightly golf this proof and the one below.

Reply:
added, love me some more dot notation! not sure how to golf the second one but they're both one-liners so ¯\_(ツ)_/¯